docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,118 @@
+
+2. Will your installation run on mainframe (z/OS, USS)?
+   1. Yes, all Zowe server components will be hosted on the mainframe.
+   2. No, all Zowe server components will be installed off-platform (z/OS).
+   3. Hybrid approach - some Zowe server components will be installed on-platform and some will be installed off-platform. 
+
+3. What certificates storage type you plan to use? 
+   1. I plan to use z/OS keyring
+   2. I plan to use keystore/truststore files pair
+
+4. Was the certificate already imported to your keystore/truststore (regardless of the certificate format and storage type)
+   1. Yes, my valid certificates were previously imported
+   2. No, I need to import my new certificates
+
+5. Do you intend to use your certificate for server, for client, or for both?
+   1. Server only
+   2. Client only
+   3. Server and Client
+   
+## Certificates configuration decision flow
+Review the diagram below to understand the certificates configuration decision flow.
+Use the answers you provided in the questionnaire above to decide which path to follow in corresponding decision block (the yellow diamonds).
+
+![Certificates configuration decision tree](../images/install/config-certificates.png)
+
+1. If you have an existing certificate, you can import it to a key storage of type depending on the certificates format.
+
+** Note: ** Before importing your certificates, check the next question to make sure that their format, type and properties correspond to the required protection and acceptability, according to the planned deployment environment (DEV, TEST, PROD).
+For example, you should not use self-signed certificates for production environments.
+
+For more information, see [Import and configure an existing certificate](./import-certificates.md).
+
+2. If your existing certificates are self-signed (Q2) and your target environment is production (Q3), we strongly recommend that you acquire new certificates from your trusted CA.
+
+3. If you do not have an existing certificate, you can create one (self-signed option) or acquire a new one from a trusted CA, depending on your target environment type (DEV/TEST or PROD).
+
+4. If you plan to use z/OS keyring you'd need to generate JCEKS type of certificate. On contrary, if you prefer to store your certificates in a keystore/truststore pair, you'd need to generate PKCS12 type of certificate.
+
+For more information, see [Generate a certificate if you do not have a certificate](./generate-certificates.md).s
+
+** Note: ** If you plan for production deployment and need to acquire certificates from a trusted CA, follow the same rule to decide what type of certificate to request from the CA.
+
+5. If you plan to use the same certificate for client and server usage - Q5 and Q6 (your service accepts inbound connections and performs calls to other TLS-secured services), you'd need to generate your certificates with the EXTENDED USAGE attribute set to CLIENT and SERVER.
```

Comment:
This is confusing, they should not be choosing this as they don't know how Zowe services are using certificates.

Reply:
Related and answered in another note above.